pygem/params.py

```diff
@@ -294,6 +299,89 @@ def print_info(self):
 		print '\nposition_vertex_3 ='
 		print self.position_vertex_3
 
+	def build_bounding_box(self, shape, tol=1e-6, triangulate=False, triangulate_tol=1e-1):
```

Comment:
You are right it is not tested. However, all the methods called inside it are. I can either repeat the `asserts` called in other thest or try `assert_called` tests